frontend/web/src/app/services/customer/customer.service.ts

```diff
@@ -43,6 +45,28 @@ export class CustomerService {
       });
   }
 
+  getDigitalParts(id: number): Observable<DigitalPart[]> {
+    return this.client.get(this.endpoint + '/' + id + '/digital-parts')
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)